Makefile

```diff
@@ -108,7 +108,10 @@ code/compile:
 .PHONY: code/gen
 code/gen:
 	operator-sdk generate k8s
-	operator-sdk generate openapi
+	operator-sdk generate crds
+	# This is a copy-paste part of `operator-sdk generate openapi` command (suggested by the manual)
```

Comment:
@slaskawi Where are you seeing that it is suggested by the manual? It was deprecated in favour of `generate crds` in operator-sdk version `0.13`[1], it does say to use it directly for OpenAPI code generation but I don't believe that is needed in this operator.

[1] https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#notable-changes

Reply:
Here's my output of executing `$ operator-sdk generate openapi` command:

```
[Deprecation notice] The 'operator-sdk generate openapi' command is deprecated!

 - To generate CRDs, use 'operator-sdk generate crds'.
 - To generate Go OpenAPI code, use 'openapi-gen'. For example:

      # Build the latest openapi-gen from source
      which ./bin/openapi-gen > /dev/null || go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen

      # Run openapi-gen for each of your API group/version packages
      ./bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/<group>/<version> -O zz_generated.openapi -p ./pkg/apis/<group>/<version> -h ./hack/boilerplate.go.txt -r "-"

INFO[0000] Running OpenAPI code-generation for Custom Resource group versions: [keycloak:[v1alpha1], ] 
INFO[0008] Code-generation complete.                    
INFO[0008] Running CRD generator.                       
INFO[0008] CRD generation complete.
```

So it seems replacing `operator-sdk generate openapi` requires using `openapi-gen` command.

Reply:
I see it here after looking - https://github.com/operator-framework/operator-sdk/blob/76b9041029e3d3d19d95ee32afebacdd8999c37a/doc/sdk-cli-reference.md#openapi

I think but maybe wrong that since all the validation is now done through the CRD itself, the `zz_generated.openapi.go` file is no longer needed but this is just a hunch. I would be curious with a brand new operator, if you added an api and ran the generation, would it generate this file at all.

Reply:
From my understanding, the Open API stuff is needed to expose and document API Server REST endpoints for our CRDs. If my train of thought is right, the `zz_generated.openapi.go` would still be needed. But I guess we need to ask some OpenShift folks to clarify this.

As for now, I would like to preserve this file. Maybe we'll kill it in subsequent release but I would like to double check if we don't break anything.

Reply:
Seems reasonable @slaskawi .